src/main/java/no/nav/tag/kontakt/oss/Kontaktskjema.java

```diff
@@ -23,7 +23,7 @@
     private String kommune;
     private String kommunenr;
     private String bedriftsnavn;
-    private String bedriftsnr;
+    private String orgnr;
```

Comment:
Hvis vi hadde vært i en reell prod-situasjon (og hatt en del trafikk) burde vi gjort denne endringen i to steg, der vi først legger til orgnr og deretter (i en ny prodsetting) fjerner bedriftsnr. Når vi fjerner ting fra datamodellen vil vi kunne få en del feil fra de instansene som ikke er oppdatert enda. Vi trenger ikke tenke på det nå, siden vi reelt sett ikke er i prod, men bør ha sånne vurderinger i bakhodet til senere :)

Reply:
Skjønner!